rust-sdk/Cargo.toml

```diff
@@ -21,3 +21,8 @@ name = "simple"
 [dev-dependencies]
 chrono = "0.4.38"
 env_logger = { version = "0.11.3", features = ["unstable-kv"] }
+
+[build-dependencies]
+gh-workflow = "0.3.0"
```

Comment:
A dependency in this package has a MSRV of `1.75` which blocks us from using it: https://github.com/JelteF/derive_more/pull/389

I will fiddle around with it to see if we can overcome the blocker.